### PR TITLE
remove unnessary usage of execution context

### PR DESCRIPTION
**Pull request by octonato**

### CODE REVIEW

**Review by raboof — APPROVED** [review]
I guess this was here not because it was needed, but to show you can use dependency injection to obtain a `Materializer` (and thus for example an ExecutionContext).

I agree with the change, but perhaps add a comment explaining the materializer is injected and ready to be used for this?

**Comment by marcospereira** [comment]
@raboof added a comment. :-)

File: app/routers/HelloWorldRouter.scala [hunk 1]
```diff
@@ -7,10 +7,15 @@ import javax.inject.Inject
 import scala.concurrent.Future
 
 class HelloWorldRouter @Inject() (mat: Materializer) extends example.myapp.helloworld.grpc.AbstractGreeterServiceRouter(mat) {
-  private implicit val ec = mat.executionContext
-
+  
+  // We need to inject a Materializer since it is required by the abstract
```
Comment (raboof) [inline app/routers/HelloWorldRouter.scala:11]: Oh I actually missed that ;)

**Review by raboof — APPROVED** [review]

**Comment by dwijnand** [comment]
Ugh.  Probably best to just find a way to need it, rather than comment it out and add a 5 line comment about it.

**Comment by octonato** [comment]
I would like to align the examples. The Akka gRPC docs have a protobuf definition with 4 methods: unary call, client stream, server stream and bi-directional.

The quickstart for Akka has only two methods and the Play and Lagom only one method. We should use the same protobuf definition all over the place. 

When we do it, we can revise that method and re-consider to add the execution context. Btw, the Lagom one doesn't have an execution context either.

**Pull request merged**